### PR TITLE
Add basic support for EMC Avamar

### PR DESCRIPTION
**Pull request by kai7p**
With these patches users can restore systems, which use the AvamarClient package and a compatible server to backup the file sytem, with ReaR. Use "BACKUP=AVA" in the local.conf or site.conf and follow the instructions.

### CODE REVIEW

**Comment by jsmeix** [comment]
@kai7p 
I know nothing at all about EMC Avamar and how it works
so that I can only make a general comment:
In restore/AVA/default/200_prompt_user_to_start_restore.sh
you call /opt/avamar/bin/avagent.bin
If /opt/avamar/bin/avagent.bin is an interactive program,
see "What to do with stdout and stderr" in
https://github.com/rear/rear/wiki/Coding-Style
basically any program that behaves interactive
or could sometimes be interactive must be called as
<pre>
COMMAND ... 0<&6 1>&7 2>&8
</pre>

File: usr/share/rear/restore/AVA/default/200_prompt_user_to_start_restore.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# the user has to do the main part here :-)
+#
+#
+/opt/avamar/bin/avagent.bin --bindir="/opt/avamar/bin" --vardir="/opt/avamar/var" --sysdir="/opt/avamar/etc" --logfile="/tmp/avagent.log"
```
Comment (gozora) [inline usr/share/rear/restore/AVA/default/200_prompt_user_to_start_restore.sh]: Why not use something like:
```
$AVA_ROOT_DIR/bin/avagent.bin --bindir="$AVA_ROOT_DIR/bin" --vardir="$AVA_ROOT_DIR/var" --sysdir="$AVA_ROOT_DIR/etc" --logfile="/tmp/avagent.log"
```
?

Comment (kai7p) [reply usr/share/rear/restore/AVA/default/200_prompt_user_to_start_restore.sh]: Good idea, change commited.

**Comment by gdha** [comment]
@kai7p Is this a partial PR? As the restore is a request to restore via a WebGUI? Are there plans to foresee a full cycle of automatic restore via `rear recover` ?

**Comment by kai7p** [comment]
@gdha I don't think Avamar offers the required APIs to automate the recovery

**Pull request merged**